### DISCUSSION TITLE
Pull request review from notification shows lots of gray

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44233] [Mobile] [answered]
GitHub Mobile beta
Open notifications
Select a review notification:
https://github.com/PowerDNS/pdns/pull/12342#pullrequestreview-1224070499

See this:

Be very confused.
![Screenshot_20230113-101937.png](https://user-images.githubusercontent.com/2119212/212355478-84400c97-821f-4c9d-9a8e-592212503b23.png)

Scroll down a bit:
![Screenshot_20230113-101950.png](https://user-images.githubusercontent.com/2119212/212355479-0d4d04c7-b50f-407b-95b6-fbb669e94695.png)
Scroll down more:
![Screenshot_20230113-102002.png](https://user-images.githubusercontent.com/2119212/212355480-b19d2785-8a03-46a7-aa63-71fceb27780d.png)

Refreshing doesn't change stuff.

### Expected results

Probably code instead of lots of gray space

Here's what it looks like on Chrome Android:
![Screenshot_20230113-105456.png](https://user-images.githubusercontent.com/2119212/212362745-835105f2-3ae2-4daf-88ec-2ae86cf6ef45.png)

**Comment by stevepopovich** [comment]
Hi @jsoref does this happen consistently for you? I am having trouble reproducing this.

Please confirm this is happening from the notification screen and not from an external deeplink?

**Reply by jsoref** [reply]
Doesn't seem to be. I'm not sure what other variables are in play.

**Reply by stevepopovich** [reply, marked as answer]
I will file a bug for this to be investigated. Thanks!
